app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -209,19 +209,39 @@
    :display :flex
    :align-items :center})
 
+(defn- vectorio-link-style
```

Comment:
Why does this link function differently than other links? If we normally have text-decoration underline and on hover it's removed. And in this link it's the otherway around?

Reply:
Because there is a comment to this jira https://jira.mkm.ee/browse/TEET-1256?focusedCommentId=1008741&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1008741, so it is implement exactly as the link to THK Project (see https://github.com/solita/mnt-teet/blob/master/app/frontend/src/cljs/teet/ui/common.cljs#L206-L208)

Reply:
Maybe this could be asked from the designers if both of these links should just work the way other links in the system work, instead of having these weird special cases.

Reply:
will double-check

Reply:
fixed